client/my-sites/activity/filterbar/date-range-selector.jsx

```diff
@@ -212,15 +216,65 @@ export class DateRangeSelector extends Component {
 		return this.getToDate( filter );
 	};
 
-	render() {
-		const { translate, isVisible, onButtonClick } = this.props;
+	renderDateRangeSelection = () => {
+		const { translate } = this.props;
 		const from = this.getFromDate();
 		const to = this.getToDate();
 		const enteredTo = this.getEnteredToDate();
 		const modifiers = { start: from, end: enteredTo };
 		const disabledDays = [ { after: new Date() } ];
 		const selectedDays = [ from, { from, to: enteredTo } ];
 
+		return (
+			<div className="filterbar__date-range-wrap">
+				<DatePicker
+					fromMonth={ this.getFromDate() }
+					selectedDays={ selectedDays }
+					disabledDays={ disabledDays }
+					modifiers={ modifiers }
+					onSelectDay={ this.handleDayClick }
+					onDayMouseEnter={ this.handleDayMouseEnter }
+				/>
+				<div className="filterbar__date-range-selection-info">
+					<div className="filterbar__date-range-info">
+						{ ! from &&
+							! to &&
+							translate( '{{icon/}} Please select the first day.', {
+								components: { icon: <Gridicon icon="info" /> },
+							} ) }
+						{ from &&
+							! to &&
+							translate( '{{icon/}} Please select the last day.', {
+								components: { icon: <Gridicon icon="info" /> },
+							} ) }
+						{ from &&
+							to && (
+								<Button borderless compact onClick={ this.handleResetSelection }>
+									{ translate( '{{icon/}} clear', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( '{{icon/}} clear dates' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).